lasagne/tests/layers/test_helper.py

```diff
@@ -4,7 +4,7 @@
 import theano
 
 
-class TestGetAllLayers:
+class TestGetAllLayers(object):
```

Comment:
Hmm, most of the tests have old-style classes... we should probably be consistent there. Is there any reason for using one or the other with pytest?